src/recordpage.h

```diff
@@ -92,13 +95,17 @@ private slots:
     void updateRecordType();
     void appendOutput(const QString& text);
     void setError(const QString& message);
+    void addKonsoleWidget();
 
     QScopedPointer<Ui::RecordPage> ui;
 
     PerfRecord* m_perfRecord;
     QString m_resultsFile;
     QElapsedTimer m_recordTimer;
     QTimer* m_updateRuntimeTimer;
+    KParts::ReadOnlyPart* m_konsolePart;
+    KService::Ptr m_service;
```

Comment:
remove (see .cpp)

Reply:
I can remove the m_service but the m_konsolePart is required to clear the terminal before rerunning perf record

Reply:
yes, that's fine - I only mean the m_service here